### PR TITLE
feat : Add Modal component

### PR DESCRIPTION
**Pull request by viru2001**
# Modal Component

## Description 

I created two types of modal : 
1. Simple modal 
2. List modal

Output : 

modal

[See preview
](https://deploy-preview-7--cool-ui.netlify.app/pages/docs#modal)

### CODE REVIEW

File: components/Modal/modal.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Modal</title>
```
Comment (amirnathani9) [inline components/Modal/modal.html:7]: here in  meta can add author and discription. Its only suggestion.

Comment (viru2001) [reply components/Modal/modal.html:7]: I will add it in my documentation page

**Review by amirnathani9 — COMMENTED** [review]
Your code is clean, clear and easy to read.You use best practices.It looks good.

File: components/Modal/modal.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+simpleModalDemoBtn.addEventListener("click", () => {
+  simpleModalWrapper.style.display = "flex";
+});
+
+listModalDemoBtn.addEventListener("click", () => {
+  listModalWrapper.style.display = "flex";
+});
+actionBtns.forEach(btn => {
+  btn.addEventListener("click", () => {
+    if (btn.parentNode.id === "simple-modal-buttons") {
```
Comment (arjunbharti) [inline components/Modal/modal.js]: You can add comments here for better code readability.

**Review by arjunbharti — COMMENTED** [review]
Added a little suggestion. Everything looks good. You can go ahead!

File: components/Modal/modal.js (hunk 2 above)
Comment (viru2001) [reply components/Modal/modal.js]: done

**Pull request merged**